src/main/java/life/qbic/model/download/QbicDataDownloader.java

```diff
@@ -49,10 +49,10 @@ public class QbicDataDownloader {
   private static final int DEFAULT_DOWNLOAD_ATTEMPTS = 3;
   private final String outputPath;
 
-  private final ChecksumReporter checksumReporter =
-          new FileSystemWriter(
-                  Paths.get(System.getProperty("user.dir") + File.separator + "logs" + File.separator + "summary_valid_files.txt"),
-                  Paths.get(System.getProperty("user.dir") + File.separator + "logs" + File.separator + "summary_invalid_files.txt"));
+  private final Path logDirectory = Paths.get(System.getProperty("user.dir"),"logs");
+  private final ChecksumReporter checksumReporter = new FileSystemWriter(
+      Paths.get(logDirectory.toString(), "summary_valid_files.txt"),
+      Paths.get(logDirectory.toString(), "summary_invalid_files.txt"));
```

Comment:
Constants could also go in the aforementioned static configuration class of the application